docs/docs/rest-api/public/api/v2/groups.raml

```diff
@@ -0,0 +1,376 @@
+
+
+    If there are no changes to the application definition, no restart is triggered.
+    During restart marathon keeps track, that the configured amount of minimal running instances are _always_ available.
+
+    This method alows 2 special modes for the update operation>
+
+    * Provide only the `version` field in the group definition. This will rollback the group to that given version
+
+    * Provide only the `scaleBy` field will scale all transitive applications by the given factor.
+
+    A deployment can run forever. This is the case, when the new application has a problem and does not become healthy.
+    In this case, human interaction is needed with 2 possible choices
+
+
+    * Rollback to an existing older version
+
+    * Update with a newer version of the group which does not have the problems of the old one.
+
+
+    If there is an upgrade process already in progress, a new update will be rejected unless the force flag is set.
+    With the force flag given, a running upgrade is terminated and a new one is started.
+
+    Since the deployment of the group can take a considerable amount of time, this endpoint returns immediatly with a version.
+    The failure or success of the action is signalled via event. There is a group_change_success and group_change_failed with
+    the given version.
+  is: [ secured ]
+  body:
+    application/json:
+      example: !include examples/group.json
+      schema: Group
+  responses:
+    200:
+      description:
+        A deployment is started which has a unique deployment identifier.
+        The related deployment can be fetched from the /v2/deployments endpoint.
+        If the deployement is gone from the list of deployments, than this means it is finished.
+        As long as the deployment runs, the effect of that change operation is visible only partially.
+      body:
+        application/json:
+          example: !include examples/deployments_result.json
+    400:
+      description: The group definition provided in the body is not valid.
+      body:
+        application/json:
+          example: |
+            {"message":"Invalid JSON","details":[{"path":"/id","errors":["error.expected.jsstring"]}]}
+    409:
+      description: There is an already deployed group with this name
+      body:
+        application/json:
+          example: |
+            {"message":"Group / is already created. Use PUT to change this group."}
+    422:
+      description: The entity send can not be preocessed, since there are validation errors
+      body:
+        application/json:
+          example: |
+            {
+              "message":"Bean is not valid",
+              "errors": [
+                {
+                  "attribute":"apps[0].id",
+                  "error":"identifier /app is not child of /group. Hint: use relative paths."
+                }
+              ]
+            }
+
+
+###
+# Create a new application group
+#
+post:
```

Comment:
The underlying implementation has a `force` flag that is undocumented but will allow changing the group even if there's a running deployment.